simpledotorg/simple-server
# 12289 patient list email to download

**Story card:** [https://app.shortcut.com/simpledotorg/story/12289/allow-users-to-download-the-patient-line-list-asynchronously](URL)

## Because

We currently send emails to users with the linelist as a csv. These linelists can be large than 25MB (which exceeds email attachment limit)

## This addresses

Removal of sidekiq job to send the report as email, and provide a zipped download stream of the report.
New environment variable "REPORT_ENUMERATOR_BATCH_SIZE" is added, which will control number of records to flush at a time to the stream.

## Test instructions

Login to the Simple App as admin.
Navigate to Reports. Click on any district.
Select "Patient Line List" from the Downloads tab on the top right corner.
It should start downloading the report
